### ISSUE TITLE
Possible typo in PlatformCompatibilityAnalyzer.cs?

### ISSUE THREAD

**Issue by omajid** [dotnet/roslyn-analyzers#7019]
https://github.com/dotnet/roslyn-analyzers/blob/91fda6d8a634d5f4715b70a24538ab52ca2a8758/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs#L1730

Should the `SupportedFirst` here be `SupportedSecond`?

**Comment by buyaa-n** [comment]
> Should the SupportedFirst here be SupportedSecond?

No, it is comparing the called API's `SupportedFirst` with the call site `SupportedFirst` and `SupportedSecond`. The called API's attribute create deny list, that means there will be no `SupportedSecond`. 
https://github.com/dotnet/roslyn-analyzers/blob/91fda6d8a634d5f4715b70a24538ab52ca2a8758/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs#L1530-L1536
We explicitly set `SupportedSecond` to null for deny list:
https://github.com/dotnet/roslyn-analyzers/blob/91fda6d8a634d5f4715b70a24538ab52ca2a8758/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs#L2024-L2029

**Comment by omajid** [comment]
Thanks!
